dream-big-ui/src/app/app.module.ts

```diff
@@ -5,6 +5,7 @@ import { BrowserModule } from '@angular/platform-browser';
 import { HttpClientModule } from '@angular/common/http';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterModule } from '@angular/router';
+import { MatDialogModule } from '@angular/material/dialog';
```

Comment:
Material modules are imported in the NgMaterial file. MatDialog is already imported there so doing it here isn't necessaey

Reply:
Okay. I have updated the pull request to add some changes to the display for the intro-page and have also removed the import for MatDialogModule.